extensions/standard-processors/processors/GenerateFlowFile.cpp

```diff
@@ -87,7 +87,7 @@ void generateData(std::vector<char>& data, bool textData = false) {
     std::uniform_int_distribution<> distr(0, index_of_last_char);
     auto rand = std::bind(distr, eng);
     std::generate_n(data.begin(), data.size(), rand);
-    std::for_each(data.begin(), data.end(), [](char & c) { c = TEXT_CHARS[c];});
+    std::for_each(data.begin(), data.end(), [](char & c) { c = TEXT_CHARS[static_cast<uint8_t>(c)];});
```

Comment:
Minor, but this could be moved to `rand`.

Reply:
Sorry I'm not sure what do you mean here. Could you give a code example/suggestion?

Reply:
Updated in 2d4d00f390e68a1128e5e9964b988c76fe3c4505